front/pages/edit.vue

```diff
@@ -95,6 +146,21 @@
         :open="openSuccessEditModal"
         >La couleur a été changée avec succès !</AppAlert
       >
+      <AppAlert
```

Comment:
Found a z-index issue, the color picker's hue cursor is drawn over the modal 
![image](https://user-images.githubusercontent.com/35401485/137507948-0bf47267-b73c-4f00-ade1-4bbb1d2b4c4f.png)